pyproject.toml

```diff
@@ -2,6 +2,84 @@
 requires = ["setuptools", "wheel"]
 build-backend = "setuptools.build_meta"
 
+[tool.setuptools.packages.find]
+where = ["src"]
+
+[project]
+name = "pyramid"
+version = "2.1.dev0"
+description = "The Pyramid Web Framework, a Pylons project"
+
+authors = [
+  {name = "Chris McDonough, Agendaless Consulting", email = "pylons-discuss@googlegroups.com"},
+]
+
+# license = "BSD-derived (Repoze)"
+license-files = ["LICENSE.txt"]
+
+readme = {file = "README.rst", content-type = "text/x-rst"}
+
+requires-python = ">=3.9"
+dependencies = [
+    "hupper >= 1.5",  # ignore_files support
+    "plaster",
+    "plaster_pastedeploy",
+    "setuptools",
+    "translationstring >= 0.4",  # py3 compat
+    "venusian >= 1.0",  # ``ignore``
```

Comment:
These inline comments are not very clear

Reply:
I copy and pasted those from the setup.py -- they should be reviewed an updated for the present needs, but I didn't feel qualified to do that.